src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -29,14 +35,19 @@ object SharedFileSystem {
     case unsupported => throw new UnsupportedOperationException(s"Localization strategy $unsupported is not recognized")
   })
 
+  val DockerLocalizers = localizePathAlreadyLocalized _ +: (LocalizationStrategies collect {
+    case "hard-link" => localizePathViaHardLink _
+    case "copy" => localizePathViaCopy _
+  })
```

Comment:
I can add a comment above `DockerLocalizers` saying that any non-recognized localizer has already been raised up by `Localizers` though.